### DISCUSSION TITLE
Update docs to show how to use the new `queryoptions` with Svelte

### DISCUSSION THREAD

**Discussion by frederikhors** [TanStack/query#6449] [Q&A]
### Describe the bug

After https://github.com/TanStack/query/pull/6211 I don't understand how to correctly use `queryOptions` with Svelte and it's reactivity mode.

Can you please update docs?

### Tanstack Query adapter

svelte-query

### TanStack Query version

5

**Comment by TkDodo** [comment]
@lachlancollins FYI. there are no tests for `queryOptions` in svelte that I could refer to
